src/atomic_structure.hh

```diff
@@ -255,15 +216,27 @@ namespace rascal {
       return is_similar;
     }
 
+    inline bool is_identical(const PositionsInput_t & positions,
+                             const AtomTypesInput_t & atom_types,
+                             const Eigen::Ref<const Eigen::MatrixXd> cell,
+                             const PBCInput_t & pbc,
+                             const double & skin2) const {
+      auto is_a_center_atom = ArrayB_t::Ones(atom_types.size());
+      return this->is_identical(positions, atom_types, cell, pbc,
+                                is_a_center_atom, skin2);
+    }
+
     inline bool is_identical(const PositionsInput_t & positions,
                              const AtomTypesInput_t & /*atom_types*/,
                              const Eigen::Ref<const Eigen::MatrixXd> cell,
                              const PBCInput_t & pbc,
+                             const ArrayB_ref & is_a_center_atom,
                              const double & skin2) const {
       bool is_similar{true};
```

Comment:
identical in the finite number world is relative to the threshold to me so identical and similar are synonyms in this context

Reply:
the function body and signature to test if identical would be the same (probably the threshold would be smaller) so that is the same to me. The fact that we use it to choose to rebuild or not the NL is not part of the function but how it is used.